tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "importHelpers": true,
+    "noEmitHelpers": true,
```

Comment:
I don't think  that you need `noEmitHelpers`, `importHelpers` should be sufficient.